service/internal/cloudconfig/template/decrypt_keys_assets_script.go

```diff
@@ -1,6 +1,8 @@
 package template
 
 const DecryptKeysAssetsScript = `#!/bin/bash -e
+set -o errexit
```

Comment:
would this be the same as if you would set `set -e`

Reply:
True, but this is at least little bit more decriptive

Reply:
Thanks, I haven't seen the longer version of it, that's why I was asking 👍